pkg/kubelet/cm/helpers_linux.go

```diff
@@ -167,6 +167,10 @@ func ResourceConfigForPod(pod *v1.Pod, enforceCPULimits bool, cpuPeriod uint64,
 	// determine the qos class
 	qosClass := v1qos.GetPodQOS(pod)
 
+	if !enforceCPULimits {
+		cpuQuota = int64(-1)
+	}
+
```

Comment:
how is this not duplicated from https://github.com/kubernetes/kubernetes/pull/117030/files#diff-060a703e5435160e728644c2e1d80ea15883254c0492308eef102ab35d1fae97R163 ?

Reply:
A rebase artifact, thanks, I will fix it.